fastlane/lib/fastlane/actions/onesignal.rb

```diff
@@ -17,6 +17,7 @@ def self.run(params)
         apns_p12_password = params[:apns_p12_password]
         android_token = params[:android_token]
         android_gcm_sender_id = params[:android_gcm_sender_id]
+        organization_id = params[:organization_id].to_s.strip
```

Comment:
Calling `.to_s.strip` would never allow `organization_id` to be `nil` so I think we need to remove this 😊 

```suggestion
        organization_id = params[:organization_id]
```